text/refactor_refman.md

```diff
@@ -17,9 +17,11 @@ contribute code enhancements and improvements to Coq.
 
 We believe the documentation should be easily understood by users who
 have only a bachelor's degree in Computer Science or Mathematics.
-It should be possible to learn and master Coq by reading the reference
-manual.  On the other hand, the reference manual shouldn't try to
-be a tutorial or a textbook.
+Users who are learning Coq should be able to use the manual as a
+reference and exhaustive documentation without having to become expert
+first. Existing online ressources should be enough to learn Coq, and
+the reference manual should be a good complement to one of the
+standard tutorials.
```

Comment:
How about something along these lines?  (as long as we drop "Existing online ressources should be enough to learn Coq")

Those who are learning Coq should be able to use the manual as an authoritative reference without becoming an expert first.  The manual complements existing tutorials; tutorials are good for a first exposure to Coq while the manual covers all the details.